packages/types/api.d.ts

```diff
@@ -2128,47 +2130,38 @@ export interface IParserErrorMessageProvider {
 }
 
 /**
- * @experimental This API is not finalized yet and may be subject to breaking changes.
+ * @experimental
  */
-export declare class LLkLookaheadStrategy implements ILookaheadStrategy {
-  readonly maxLookahead: number
-  constructor(options: { maxLookahead?: number })
-  validate(options: {
-    rules: Rule[]
-    tokenTypes: TokenType[]
-    grammarName: string
-  }): ILookaheadValidationError[]
+export interface ILLKLookaheadValidator {
```

Comment:
We should try to keep it consistent, either `LLK` or `LLk`. I've used the latter, but I'm fine with the former as well.

```suggestion
export interface ILLkLookaheadValidator {
```

Reply:
Thanks that was not intentional